			     BASH PATCH REPORT
			     =================

Bash-Release:	5.2
Patch-ID:	bash52-020

Bug-Reported-by:	Dima Korobskiy <dkroot2@gmail.com>
Bug-Reference-ID:	<16664c2d-40ec-df33-b932-83db06e39a82@gmail.com>
Bug-Reference-URL:	https://lists.gnu.org/archive/html/bug-bash/2023-08/msg00125.html

Bug-Description:

The parser did not allow `time' to appear as the first reserved word in a
command substitution.

Patch (apply with `patch -p0'):

*** ../bash-5.2-patched/parse.y	Tue Dec 13 12:53:21 2022
--- parse.y	Fri Sep  1 10:36:28 2023
***************
*** 3151,3154 ****
--- 3151,3155 ----
      case TIMEOPT:	/* time -p time pipeline */
      case TIMEIGN:	/* time -p -- ... */
+     case DOLPAREN:
        return 1;
      default:
*** ../bash-5.2-patched/y.tab.c	Tue Dec 13 12:53:21 2022
--- y.tab.c	Fri Sep  1 10:36:44 2023
***************
*** 5466,5469 ****
--- 5466,5470 ----
      case TIMEOPT:	/* time -p time pipeline */
      case TIMEIGN:	/* time -p -- ... */
+     case DOLPAREN:
        return 1;
      default:

*** ../bash-5.2/patchlevel.h	2020-06-22 14:51:03.000000000 -0400
--- patchlevel.h	2020-10-01 11:01:28.000000000 -0400
***************
*** 26,30 ****
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 19
  
  #endif /* _PATCHLEVEL_H_ */
--- 26,30 ----
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 20
  
  #endif /* _PATCHLEVEL_H_ */
